detectron2/export/c10.py

```diff
@@ -74,7 +74,13 @@ def has(self, name):
         return name in self.batch_extra_fields
 
     def set(self, name, value):
-        data_len = len(value)
+        # len(tensor) leads to constants during tracing mode
```

Comment:
Please comment that the logic is to avoid warning and not for correctness, if that's the case.